test/browser_tests/spec_suites/newsroom-press-resources.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+var PressResources = require(
+    '../page_objects/page_newsroom-press-resources.js'
+  );
+
+describe( 'The Newsroom Press Resources Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new PressResources();
+    page.get();
+  } );
+
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'Press resources' );
+  } );
+
+  it( 'should include a main title', function() {
+    expect( page.mainTitle.getText() ).toBe( 'Press Resources' );
+  } );
+
+  it( 'should have a side nav', function() {
+      expect( page.sideNav.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include a sub title', function() {
+    expect( page.subTitle.getText() ).toBe( 'Press contacts' );
+  } );
+
+  it( 'should include a Press contact email address', function() {
+    expect( page.contactListEmail .getText() )
+    .toBe( 'press@consumerfinance.gov' );
+    expect( page.contactListEmail .getAttribute( 'href' ) )
+    .toBe( 'mailto:inquiries@consumerfinance.gov' );
+  } );
+
+  it( 'should include a Press contact list phone number', function() {
+    expect( page.contactListPhone.getText() ).toBe( '(202) 435-7170' );
+    expect( page.contactListPhone.getAttribute( 'href' ) )
+    .toBe( 'tel:2024357170' );
+  } );
+
+  it( 'should include a Press section title', function() {
+    expect( page.pressSectionTitle.getText() ).toBe( 'Photos and bios' );
+  } );
+
+  it( 'should include a Press section intro', function() {
+    expect( page.pressSectionIntro.isPresent() ).toBe( true );
+  } );
+
+  it( 'should include Director’s bio', function() {
+    expect( page.directorsImage.getAttribute( 'src' ) ).toBe(
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/071c9fc3f5c87f92f74c956a1b34fa0c0d665156